compiler/qsc_doc_gen/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "qsc_doc_gen"
+
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+homepage.workspace = true
+license.workspace = true
+repository.workspace = true
+
+[dev-dependencies]
+expect-test = { workspace = true }
+indoc = { workspace = true }
+
+[dependencies]
+enum-iterator = { workspace = true }
+regex-lite = { workspace = true }
+async-trait = { workspace = true }
+chrono = { workspace = true }
+clap = { workspace = true, features = ["derive", "cargo"] }
+env_logger = { workspace = true }
+log = { workspace = true }
+miette = { workspace = true, features = ["fancy"] }
+num-bigint = { workspace = true }
+num-complex = { workspace = true }
+qsc_codegen = { path = "../qsc_codegen" }
+qsc_data_structures = { path = "../qsc_data_structures" }
+qsc_eval = { path = "../qsc_eval" }
+qsc_frontend = { path = "../qsc_frontend" }
+qsc_ast = { path = "../qsc_ast" }
+qsc_fir = { path = "../qsc_fir" }
+qsc_hir = { path = "../qsc_hir" }
+qsc_passes = { path = "../qsc_passes" }
+qsc_project = { path = "../qsc_project", features = ["fs"] }
+rustc-hash = { workspace = true }
+thiserror = { workspace = true }
```

Comment:
Does this crate really depend on all of the above?

Reply:
I've cleaned up the Cargo.toml files for the language server and the new docs generation module.